### PR TITLE
WIP: Upgrade to 0.19

### PR DESCRIPTION
**Pull request by rtfeldman**
My goal is to:

1. Get this branch ready to go (need to update README example, among other things)
2. Backport it to 0.18
3. Publish that version as the final 0.18 release
4. Publish this as the first 0.19 release, ideally as a `PATCH` change (and if not, as close to that as possible!)

### CODE REVIEW

File: src/Css/Global.elm [hunk 1]
```diff
@@ -766,7 +689,7 @@ dl =
 
 {-| Selector for a dt element.
 
-    https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dt
+    https :// developer.mozilla.org / en - US / docs / Web / HTML / Element / dt
```
Comment (ChristophP) [inline src/Css/Global.elm]: Some white spaces seem to have snuck into the Urls

**Comment by rlefevre** [comment]
Hopefully I am doing something wrong, but something bad seems to happen with the branch `0-19-ready` and this SSCCE (found from a bigger issue reported by @n1k0 on slack):
```elm
import Html.Styled exposing (toUnstyled, a)
import Html.Styled.Attributes exposing (href)
import Html exposing (Html)

main : Html msg
main = a [ href "#" ] [] |> toUnstyled
```
I get (on Firefox 61.0.1):
```
TypeError: styles is undefined localhost:8080:5850:7
author$project$Css$Preprocess$Resolve$applyStyles<
http://localhost:8080/:5850:7
A2
http://localhost:8080/:66:24
author$project$Css$Preprocess$Resolve$expandStyleBlock
http://localhost:8080/:6054:9
author$project$Css$Preprocess$Resolve$toDeclarations
http://localhost:8080/:6097:11
author$project$Css$Preprocess$Resolve$extract
http://localhost:8080/:6070:4
author$project$Css$Preprocess$Resolve$toStructure
http://localhost:8080/:6151:21
author$project$Css$Preprocess$Resolve$compile1
http://localhost:8080/:6601:4
elm$core$List$map</<
http://localhost:8080/:4202:7
A2
http://localhost:8080/:66:24
elm$core$List$foldrHelper<
http://localhost:8080/:4147:12
A4
http://localhost:8080/:72:24
elm$core$List$foldr<
http://localhost:8080/:4192:10
A3
http://localhost:8080/:69:24
elm$core$List$map<
http://localhost:8080/:4196:10
A2
http://localhost:8080/:66:24
author$project$Css$Preprocess$Resolve$compile
http://localhost:8080/:6607:3
author$project$VirtualDom$Styled$toDeclaration
http://localhost:8080/:6645:9
author$project$VirtualDom$Styled$toStyleNode
http://localhost:8080/:6661:5
author$project$VirtualDom$Styled$unstyle<
http://localhost:8080/:6674:19
A3
http://localhost:8080/:69:24
author$project$VirtualDom$Styled$toUnstyled
http://localhost:8080/:6809:11
<anonyme>
http://localhost:8080/:6844:32
<anonyme>
http://localhost:8080/:11:2

```

I tested the branch with:
```
$ wget https://github.com/rtfeldman/elm-css/archive/0.19-ready.zip
$ unzip 0.19-ready.zip
$ elm install Skinney/murmur3
$ elm install elm/json
$ elm install rtfeldman/elm-hex
$ elm install elm/virtual-dom
```
then by adding `"elm-css-0.19-ready/src"` to the `source-directories` of my `elm.json` file.

Note that most attributes lead to a crash, so any of the following `main` function crashes:
```
main = a [ href "" ] [] |> toUnstyled

main = div [ class " ] [] |> toUnstyled

main = div [ id "" ] [] |> toUnstyled
```
The following don't:
```
main = div [ attribute "" "" ] [] |> toUnstyled

main = div [] [] |> toUnstyled
```

**Comment by rtfeldman** [comment]
@rlefevre @n1k0 fixed - try now!

**Comment by rlefevre** [comment]
Yes, it fixed it. Thanks you :+1:

**Comment by n1k0** [comment]
Yes, the runtime error is gone, thanks everybody :)

File: src/Css/Global.elm (hunk 1 above)
Comment (rtfeldman) [reply src/Css/Global.elm]: Fixed, thanks!

**Comment by dfeyer** [comment]
I have a bunch or erros in the Svg module:

```
VirtualDom.Styled.attribute "values" (VirtualDom.Styled.noJavaScriptUri value)
                                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
The `VirtualDom.Styled` module does not expose a `noJavaScriptUri` variable.
```

**Comment by rtfeldman** [comment]
@dfeyer whoops, fixed now!

**Comment by rtfeldman** [comment]
@dfeyer @n1k0 @rlefevre if you have time to try it out, https://github.com/rtfeldman/elm-css/pull/465 is what I'm planning to use as the final release tomorrow, so any feedback would be greatly appreciated! ❤️

**Comment by JariGitHubber** [comment]
I am a newbie in ELM, but I try to get Css working with Elm 0.19. I installed: Elm package install rtfeldman/elm-css. Installation ok, but then import Css is failing as a "bad import".
Looking for corrrect Css and newbie level instructions how to implement..

**Pull request merged**